assignment2/src/com/company/main/FileOperations.java

```diff
@@ -0,0 +1,43 @@
+package com.company.main;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.company.main.Details.students;
+
+public class FileOperations {
+
+    public void writeToFile() {
+        try {
+            FileOutputStream f = new FileOutputStream(new File("myObjects.ser"));
+            ObjectOutputStream o = new ObjectOutputStream(f);
+            o.writeObject(students);
+            o.close();
+            f.close();
+        }
+        catch (Exception e){
+            System.out.println("Student details couldn't write to file");
```

Comment:
we lost context about the exception here, how to identify the cause of the exception or even what is the exception from logs